client/src/Users/UsersList.js

```diff
@@ -152,7 +153,18 @@ function UsersList() {
   }
 
   function onMouseOverZip(feature) {
-    setSelectedFeature(feature);
+    if (resetTimeoutId) {
+      clearTimeout(resetTimeoutId);
+    }
+    setSelectedFeature(feature); 
+
+    
+    const newTimeoutId = setTimeout(() => {
+      setSelectedFeature(null);
+    }, 100);
+
+    setResetTimeoutId(newTimeoutId);
```

Comment:
Introducing this new state variable actually has more significant side effects than first appears. Because this function is dependent upon the variable, every time the variable is updated with the setter, it's going to trigger a redraw of the component and, because the map depends on this function, a redraw of the map.

The result is that, when the mouse is over the map, the browser enters into effectively an infinite loop of redraws until the mouse leaves the map. This is not really an efficient way to have the app run. To confirm this, add a console.log into this function. You'll see it print out repeatedly non-stop when the mouse is over a map.